application.rb

```diff
@@ -1,4 +1,10 @@
 require_relative 'product'
+require_relative 'vendor'
 
 # Creating new product
-product = Product.new("Bread", 12)
+product_bread = Product.new('Bread', 12)
+product_milk = Product.new('Milk', 23)
+
+# Creating new vendor
+product_list = [product_bread, product_milk]
```

Comment:
no need to name the variable like `product_list`
`products` has enough information to understand that it is an array.